src/ot_orchestration/dags/config/pis.yaml

```
@@ -211,6 +211,11 @@ steps:
       source: http://purl.obolibrary.org/obo/go.obo
       destination: go/go.obo
 
+  gold_standard:
```

```suggestion
  l2g_gold_standard:
```

Although if you think we might need more files downloaded for gentropy, I'd name the step `gentropy`, unless the amount becomes worthy of splitting.